dali/base/dafdesc.cpp

```diff
@@ -3208,20 +3347,36 @@ void initializeStorageGroups(bool createPlanesFromGroups)
             {
                 IPropertyTree & cur = dropzones->query();
                 unsigned numServers = cur.getCount("ServerList");
+
+                //Allow url style drop zones, and drop zones with a single node.  Not sure what >1 would mean in legacy.
                 if (numServers <= 1)
                 {
-                    //If no instances then assume it is always local (e.g. azure)
-                    const char * ip = numServers ? cur.queryProp("ServerList[1]/@server") : nullptr;
-                    IPropertyTree * plane = storage->addPropTree("planes");
-                    plane->setProp("@name", cur.queryProp("@name"));
-                    plane->setProp("@prefix", cur.queryProp("@directory"));
-
-                    //Temporary.  This is likely to change to eventually create a hostgroup and set the @hosts property.
-                    if (ip && !streq(ip, "."))
-                        plane->setProp("@host", ip);
+                    GroupInformation * next = new GroupInformation(cur.queryProp("@name"));
+                    const char * ip = cur.queryProp("ServerList[1]/@server");
+
+                    next->dir.set(cur.queryProp("@directory"));
+                    next->dropZone= true;
+                    if (ip)
+                        next->hosts.append(ip);
+                    appendGroup(allGroups, next);
                 }
             }
         }
+
+        //Sort into size order, to help spot groups that are subsets of other groups
+        allGroups.sort(compareGroupSize);
+
+        //Now generate a list of hosts, if one group is a subset of another then do not generate the hosts
+        generateHosts(storage, allGroups);
+
+        //Finally generate the storage plane information
+        ForEachItemIn(i, allGroups)
+        {
+            const GroupInformation & cur = allGroups.item(i);
+            cur.createStoragePlane(storage, 0);
+            if (cur.groupType == grp_thor)
+                cur.createStoragePlane(storage, 1);
```

Comment:
I think really this should depend on whether the ThorCluster that caused the group to be created has replication on or not.

Reply:
It shouldn't have any ill effect if it is created.